app/route_rangers_api/models.py

```diff
@@ -1 +1,143 @@
-from django.db import models
+from django.contrib.gis.db import models
+
+#################################
+###### DEMOGRAPHIC MODELS #######
+#################################
+
+
+class Demographics(models.Model):
+    """
+    Class to represent demographic data pulled from the ACS Survey
+    """
+    census_tract = models.CharField(
+        max_length=15
+    )  # Check length of census tract if its uniform to enforce it here
+    state = models.CharField(
+        max_length=15
+    )  # Check if it's worth to keep or if we should add a method
+    county = models.CharField(max_length=15)  # Same as above
+    median_income = models.IntegerField()
+    transportation_to_work = models.CharField(
+        verbose_name="Means of Transportation to Work"
+    )
+    work_commute_time = models.FloatField(verbose_name="Time of commuto to work")
+    vehicles_available = models.IntegerField()
+    disability_status = models.IntegerField(
+        verbose_name="Number of people with disability"
+    )
+
+#################################
+######## TRANSIT MODELS #########
+#################################
+
+
+class TransitRoute(models.Model):
+    """
+    Class that represent subway lines and bus routes
+    """
+    city = models.CharField(max_length=30)
+    route_id = models.CharField(max_length=30)
+    geo_representation = models.LineStringField()
+    mode = models.CharField(verbose_name="Mode of transportation", max_length=10)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["city","route_id"],name="city route id")
+        ]
+
+
+class TransitStation(models.Model):
+    """
+    Class that represents bus stops and station complex
+    (i.e. CTA - Roosevelt)
+    """
+    city = models.CharField(max_length=30)
+    station = models.CharField(max_length=30)
+    location = models.PointField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["city","station"],name="city station")
+        ]
+
+
+class StationRouteRelation(models.Model):
+    """
+    Class that represent relationship between Bus Stops/Subway Stations
+    and the routes it serve.
+    (i.e. row 1: CTA Roosevelt - Green Line
+     row 2: CTA Roosevelt - Red Line)
+    """
+    station = models.ForeignKey(TransitStation, on_delete=models.PROTECT)
+    route = models.ForeignKey(TransitRoute, on_delete=models.PROTECT)
+
+
+class RidershipRoute(models.Model):
+    """
+    Class that represent ridership at the route level
+    """
+    route = models.ForeignKey(TransitRoute, on_delete=models.PROTECT)
+    date = models.DateField()
+    ridership = models.IntegerField()
+
```

Comment:
@JPMartinezClaeys depending on the response to this question, the `TransitModes` might need to be accessible by Ridership instead of nested within `TransitRoute`